site/docs/v1/tech/apis/_import-webauthn-request-body.adoc

```diff
@@ -0,0 +1,73 @@
+==== Request Body
+
+[.api]
+[field]#credentials# [type]#[Array]# [required]#Required#::
+The list of passkeys to import.
+
+[field]#credentials``[x]``.algorithm# [type]#[Long]# [required]#Required#::
+The identifier for the signing algorithm used with the passkey. These values are defined by https://www.rfc-editor.org/rfc/rfc9053[RFC 9053]. FusionAuth supports a subset of these algorithms.
+
+[field]#credentials``[x]``.attestationType# [type]#[String]# [optional]#Optional# [default]#defaults to `none`#::
+The attestation type provided when the passkey was registered. FusionAuth cannot validate the attestation on import. This is for record keeping only.
+
+[field]#credentials``[x]``.authenticatorSupportsUserVerification# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
+Indicates whether user verification was provided at the time the passkey was registered.
```

Comment:
how about:

```
Indicates whether this authenticator supports user verification. User verification ensures that the user is authorized to use the authenticator.
+
This value is currently determined by whether user verification occurred during the registration of this passkey rather than by examining FIDO metadata.
```

Or something similar. Happy to chat sync if that would be helpful too.